uploadNeuroDB/register_processed_data.pl

```diff
@@ -465,13 +586,22 @@ sub which_directory {
 
 
 =pod
-Function that will insert into the files_intermediary table of the database, intermediary outputs that were used to obtain the processed file.
-- Input:  - fileID : fileID of the registered processed file
-          - inputFileIDs : array containing the list of input files that were used to obtain the processed file
-          - tool : tool that was used to obtain the processed file
-- Output: - return undef if insertion did not succeed
-          - return 1 if insertion into the intermediary table succeeded. 
+
+=head3 insert_intermedFiles($fileID, $inputFileIDs, $tool)
+
+Function that will insert into the C<files_intermediary> table of the database,
```

Comment:
Grammar: Function that will insert the intermediary outputs that were used to obtain the processed file into the C<files_intermediary> table of the database.